apps/testingaccessibility/src/pages/api/auth/[...nextauth].ts

```diff
@@ -57,14 +58,17 @@ export const nextAuthOptions: NextAuthOptions = {
         const {getPurchasesForUser} = getSdk()
         const purchases = await getPurchasesForUser(token.id as string)
         token.purchases = purchases
+        session.rules = await defineRulesForPurchases(purchases)
```

Comment:
at first I put the purchases on the session but [CASL recommends caching the rules on the session](https://casl.js.org/v5/en/cookbook/cache-rules) so I tried that and it worked well!

This should be fresh and gets refreshed consistently